variables.tf

```diff
@@ -28,3 +24,12 @@ variable "mz_acceptance_required" {
   default     = false
   type        = bool
 }
+
+# Empty list means only same-region access is allowed
+# For cross-region access, add the regions to the list
+# This will be the regions where your Materialize instance is deployed
+variable "mz_supported_regions" {
+  description = "The set of regions from which service consumers can access the service"
```

Comment:
Also a bit of nit:

When possible we should stick to prior work for naming... I think AWS calls this active_remote_regions?

```
description="The set of regions that will be allowed to create a privatelink connection to the MSK cluster."
```